.env

```diff
@@ -0,0 +1,2 @@
+secret=deff1952d59f883ece260e8683fed21ab0ad9a53323eca4fasdewr344sd
```

Comment:
Its a really bad practice to store sensitive config as a plain text. 
See credentials and keyvaults to know more on how to store secrets.